test/team_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'pry'
+require "./lib/player.rb"
+require "./lib/team.rb"
+require "minitest/autorun"
+require "minitest/pride"
+
+class TeamTest < Minitest::Test
+
+  def test_player_has_attributes
+    france = Team.new("France")
+
+    assert_equal "France", france.country
+    assert_equal [], france.players
+    assert_equal false, france.eliminated?
+    # assert_equal true, france.eliminated? == true
+
+  end
+
+  def test_player_can_be_added_to_team
```

Comment:
In this test, we would want to see an assertion for the players being included in the `@players` array.  It looks like the implementation for the `add_players` method works, but it is not being tested!